apps/backend/src/modules/users/user.repository.ts

```diff
@@ -1,15 +1,25 @@
-import { type Repository } from "~/libs/types/types.js";
+import {
+	type PaginationQueryParameters,
+	type Repository,
+} from "~/libs/types/types.js";
 import { UserEntity } from "~/modules/users/user.entity.js";
 import { type UserModel } from "~/modules/users/user.model.js";
 
-import { type UserPatchRequestDto } from "./libs/types/types.js";
+import {
+	type UserGetAllResponseDto,
+	type UserPatchRequestDto,
+} from "./libs/types/types.js";
 
 class UserRepository implements Repository {
 	private userModel: typeof UserModel;
 	public constructor(userModel: typeof UserModel) {
 		this.userModel = userModel;
 	}
 
+	public async count(): Promise<number> {
+		return await this.userModel.query().resultSize();
+	}
```

Comment:
```suggestion
```

Reply:
Oh yeah, thank you. Today have big problems with power outages, so I'm kind of in the rush, forgetting things😅. Fixed everything.